spec/unit/resources/fc_network/spec_delete_bulk.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './../../../spec_helper'
```

Comment:
Add copy rights in this file

Reply:
none of chef spec files have copy rights mentioned

Reply:
done